cmd/lifecycle/rebaser.go

```diff
@@ -37,6 +37,7 @@ func (r *rebaseCmd) DefineFlags() {
 	cli.FlagRunImage(&r.RunImageRef)
 	cli.FlagUID(&r.UID)
 	cli.FlagUseDaemon(&r.UseDaemon)
+	cli.FlagPreviousImage(&r.PreviousImageRef)
```

Comment:
@natalieparellano did you have thoughts on if we should guard this on a platform API version or not? It isn't a breaking change, so I'm inclined to ship it without a guard...but that's me 😄

Reply:
Aye... I'm inclined to guard it just because that's what we do elsewhere and having two different patterns might be confusing. That said, maybe it makes sense to revisit our current strategy as the strictness we exercise around forwards-compatibility might be too extreme.

Reply:
I do think that is a discussion worth having. Anything to ease the process for newcomers is a good thing IMO. But we can discuss that later.

What platform version would you want to use? I can make a PR to the spec to document the new parameter and behavior.